CHANGELOG.md

```diff
@@ -14,42 +14,61 @@
+* `vcd_vapp_vm` and `vcd_vm` resources support security tag management via new field `security_tags` ([#1006](https://github.com/vmware/terraform-provider-vcd/pull/1006), [#1046](https://github.com/vmware/terraform-provider-vcd/pull/1046))
+* Resource `vcd_nsxt_ipsec_vpn_tunnel` adds support for custom `remote_id` field and certificate
+  based auth via fields `authentication_mode`, `certificate_id`, `ca_certificate_id` ([#1010](https://github.com/vmware/terraform-provider-vcd/pull/1010))
 * `vcd_org_group` adds `OAUTH` as an option to argument `provider_type` ([#1013](https://github.com/vmware/terraform-provider-vcd/pull/1013))
+* Resource and data source `vcd_nsxt_alb_virtual_service` add support for Transparent mode in VCD
+  10.4.1+ via field `is_transparent_mode_enabled` ([#1024](https://github.com/vmware/terraform-provider-vcd/pull/1024))
+* Resource and data source `vcd_nsxt_alb_pool` add support for Pool Group Membership via field
+  `member_group_id` ([#1024](https://github.com/vmware/terraform-provider-vcd/pull/1024))
+* Support provider authentication using Active Service Accounts ([#1040](https://github.com/vmware/terraform-provider-vcd/pull/1040))
+* Resource and data source `vcd_nsxt_network_imported` support Distributed Virtual Port Group (DVPG)
+  backed Org VDC network ([#1043](https://github.com/vmware/terraform-provider-vcd/pull/1043))
 
 ### BUG FIXES
 * Fix a bug that prevented returning a specific error while authenticating provider with invalid
   password ([#962](https://github.com/vmware/terraform-provider-vcd/pull/962))
-* Add `prefix_length` field to `vcd_vapp_network` as creating IPv6 vApp networks was not supported due to the lack of a suitable subnet representation (Issue #999) ([#1007](https://github.com/vmware/terraform-provider-vcd/pull/1007))
+* Add `prefix_length` field to `vcd_vapp_network` as creating IPv6 vApp networks was not supported due to the lack of a suitable subnet representation (Issue #999) ([#1007](https://github.com/vmware/terraform-provider-vcd/pull/1007), [#1031](https://github.com/vmware/terraform-provider-vcd/pull/1031))
 * Remove incorrect default value from `vcd_vapp_network` `netmask` field, as it prevents using IPV6 networks. Users of already defined resources need to add a `netmask = "255.255.255.0"` when using Ipv4 ([#1007](https://github.com/vmware/terraform-provider-vcd/pull/1007))
 
 ### DEPRECATIONS
-* Deprecate `netmask` in `vcd_vapp_network` ([#1007](https://github.com/vmware/terraform-provider-vcd/pull/1007))
+* Deprecate `netmask` in favor of `prefix_length` for `vcd_vapp_network` ([#1007](https://github.com/vmware/terraform-provider-vcd/pull/1007))
 
 ### NOTES
-* Add missing test name fields for TestAccVcdNsxtEdgeBgpConfigIntegrationVdc and
-  TestAccVcdNsxtEdgeBgpConfigIntegrationVdcGroup ([#958](https://github.com/vmware/terraform-provider-vcd/pull/958))
+* Add missing test name fields for `TestAccVcdNsxtEdgeBgpConfigIntegrationVdc` and
```

Comment:
Is there an easy way to elaborate this, so the casual user understands what this change is about?

Reply:
It's related to the test internal working. Basically, we were missing a field that affects the binary tests, which could not run for lack of correctly built HCL script
(see PR #958)